scripts/esrupgrades/esr.5.37-7.8.mysql.cleanup.sql

```diff
@@ -0,0 +1,120 @@
+   which are not present in the script migration. We drop such rows here */
+DELETE FROM Roles WHERE Name = 'playbook_member';
+DELETE FROM Roles WHERE Name = 'playbook_admin';
+DELETE FROM Roles WHERE Name = 'run_member';
+DELETE FROM Roles WHERE Name = 'run_admin';
+DELETE FROM Systems WHERE Name = 'PlaybookRolesCreationMigrationComplete';
+DELETE FROM Systems WHERE Name = 'playbooks_manage_roles';
+DELETE FROM Systems WHERE Name = 'playbooks_permissions';
+
+/* When migrating through the server, Boards adds a row to the Systems table. We drop it here */
+DELETE FROM Systems WHERE Name = 'products_boards';
+
+/* doRemainingSchemaMigrations adds an ID to the TeamInviteId column in the Teams table. Something like:
+       Get all affected teams with SELECT * FROM Teams WHERE InviteId = ''
+       For each of the previous team, generate a newID and run UPDATE Teams SET InviteId = newID WHERE ...
+   At the end of such migration, a new row to the Systems table is added. We drop it here */
+DELETE FROM Systems WHERE Name = 'RemainingSchemaMigrations';
+
+/* doCustomGroupAdminRoleCreationMigration adds a new role and mark this migration as complete. The new role is:
+       roles[SystemCustomGroupAdminRoleId] = &Role{
+           Name:          "system_custom_group_admin",
+           DisplayName:   "authentication.roles.system_custom_group_admin.name",
+           Description:   "authentication.roles.system_custom_group_admin.description",
+           Permissions:   SystemCustomGroupAdminDefaultPermissions,
+           SchemeManaged: false,
+           BuiltIn:       true,
+       }
+
+   The permissions contain:
+       SystemCustomGroupAdminDefaultPermissions = []string{
+           PermissionCreateCustomGroup.Id,         // create_custom_group
+           PermissionEditCustomGroup.Id,           // edit_custom_group
+           PermissionDeleteCustomGroup.Id,         // delete_custom_group
+           PermissionRestoreCustomGroup.Id,        // restore_custom_group
+           PermissionManageCustomGroupMembers.Id,  // manage_custom_group_members
+       }
+   The ID of the role will change, so we need to standardize it */
+DELETE FROM Roles WHERE Name = 'system_custom_group_admin';
+DELETE FROM Systems WHERE Name = 'CustomGroupAdminRoleCreationMigrationComplete';
+
+
+/* doPostPriorityConfigDefaultTrueMigration updates the config, setting ServiceSettings.PostPriority to true,
+   and marking this migration as complete in the Systems table. The migration script will not contain this
+   migration, so it's safe to clean it up here. */
+DELETE FROM Systems WHERE Name = 'PostPriorityConfigDefaultTrueMigrationComplete';
+
+/* getAddCustomUserGroupsPermissions and getAddCustomUserGroupsPermissionRestore adds two rows to the Systems
+   table to mark those migrations as complete. The migrations are the following:
+       func (a *App) getAddCustomUserGroupsPermissions() (permissionsMap, error) {
+           t := []permissionTransformation{}
```

Comment:
nit: can we permalink to the GitHub branch where these last existed instead of embedding?

Reply:
Done! Improved docs in https://github.com/mattermost/mattermost-server/pull/22448/commits/88cad7915b0311f0f23f57573300721a30496d8a